### PR TITLE
fixes OSIO issue 1526

### PR DESCRIPTION
**Pull request by miteshvp**
fixes https://github.com/openshiftio/openshift.io/issues/1526

### CODE REVIEW

**Comment by miteshvp** [comment]
@msrb @tisnik - we are green. PTAL.

**Comment by jpopelka** [comment]
Looks like poor reviewer job in #63 :-)

**Comment by miteshvp** [comment]
actually a silly mistake from a developer in #63

File: src/graph_populator.py [hunk 1]
```diff
@@ -311,13 +312,21 @@ def create_query_string(cls, input_json):
...
+                if not prp_package:
+                    str_gremlin += "pkg = g.V().has('ecosystem','{ecosystem}')." \
+                                   "has('name', '{pkg_name}').tryNext().orElseGet{{" \
+                                   "graph.addVertex('ecosystem', '{ecosystem}', 'name', " \
+                                   "'{pkg_name}', 'vertex_label', 'Package')}};" \
+                                   "pkg.property('last_updated', {last_updated});".format(
+                                        ecosystem=ecosystem, pkg_name=pkg_name,
+                                        last_updated=str(time.time())
                                    )
+                str_gremlin += "edge_c = g.V().has('pecosystem','{ecosystem}').has('pname'," \
```
Comment (jpopelka) [inline src/graph_populator.py]: Previously this was in `if prp_package:`, so shouldn't it now be in `else:` ?

File: src/graph_populator.py [hunk 2]
```diff
@@ -311,13 +312,21 @@ def create_query_string(cls, input_json):
...
-                                   ".orElseGet{{pkg.addEdge('has_version', ver)}};".format(
-                                        ecosystem=ecosystem, pkg_name=pkg_name, version=version
+                if not prp_package:
+                    str_gremlin += "pkg = g.V().has('ecosystem','{ecosystem}')." \
+                                   "has('name', '{pkg_name}').tryNext().orElseGet{{" \
+                                   "graph.addVertex('ecosystem', '{ecosystem}', 'name', " \
+                                   "'{pkg_name}', 'vertex_label', 'Package')}};" \
+                                   "pkg.property('last_updated', {last_updated});".format(
+                                        ecosystem=ecosystem, pkg_name=pkg_name,
+                                        last_updated=str(time.time())
```
Comment (jpopelka) [inline src/graph_populator.py]: This doesn't look like a simple fix for the mistake from #63 
Is it really needed ?

File: src/graph_populator.py (hunk 1 above)
Comment (miteshvp) [reply src/graph_populator.py]: Maybe I should add more comments. Previously I was creating an edge if Package and Version both are present. With recent changes in data importer, we now enable to do selective ingestion for either Package or Version separately. What I did was, if Version is created, I also create Package Node and create an edge between them. Hope it helps.

**Pull request merged**